changelog_unreleased/api/pr-8983.md

```diff
@@ -0,0 +1,13 @@
+#### ESM standalone bundles ([#8983](https://github.com/prettier/prettier/pull/8983) by [@Monchi](https://github.com/Monchi))
+
+Now standalone is also bundled as ESM. It works directly on browser.
+
+```js
+import prettier from "prettier/standalone";
+import parserGraphql from "prettier/parser-graphql";
+
+prettier.format("query {     }", {
+  parser: "graphql",
+  plugins: [parserGraphql],
+});
+```
```

Comment:
This can't work in browser, should be something like 


```js
import prettier from "https://unpkg.com/prettier@2.0.5/standalone.mjs";
import parserGraphql from "https://unpkg.com/prettier@2.0.5//parser-graphql.mjs";

prettier.format("query {     }", {
  parser: "graphql",
  plugins: [parserGraphql],
});
```

Reply:
Ahh yes, I didn't mean that this code works on browser, but it's misleading. Fixed.